priam/src/main/java/com/netflix/priam/defaultimpl/CassandraProcessManager.java

```diff
@@ -154,6 +157,34 @@ public void stop() throws IOException {
         stopCass.redirectErrorStream(true);
 
         instanceState.setShouldCassandraBeAlive(false);
+        if (config.getGracefulDrainHealthWaitSeconds() > 0) {
```

Comment:
Can we put a force option so that operators can call C* stop immediately without waiting for the drain in *some* scenarios?